pkg/machine/shim/networking.go

```diff
@@ -11,7 +11,10 @@ import (
 	"github.com/containers/common/pkg/config"
 	gvproxy "github.com/containers/gvisor-tap-vsock/pkg/types"
 	"github.com/containers/podman/v5/pkg/machine"
+	"github.com/containers/podman/v5/pkg/machine/connection"
```

Comment:
I believe this is what broke the compilation on pretty much every architecture but arm64 and amd64.

See https://buildd.debian.org/status/package.php?p=libpod&suite=experimental for build logs:

```
package github.com/containers/podman/pkg/machine/shim
	imports github.com/containers/podman/v5/pkg/machine/connection: build constraints exclude all Go files in /<<PKGBUILDDIR>>/_output/src/github.com/containers/podman/pkg/machine/connection
```

Is `pkg/machine/shim` supposed to be compiled on those architectures or not? 
cc: @n1hility  @mheon @baude @vrothberg

Reply:
FYI: I suggest you create a new issues instead of commenting on old PRs.

That said podman compiles just fine on other arches as well, the issue how your debian build tooling works.  As machine is only compiled on amd64 and arm64 this package is not in the import chain and as such will not cause any compile errors. Your build chain is breaking because it calls something like go build ./pkg/machine/shim which will then of course fails on the other architectures as it is not designed to build there.

It is basically the same as https://github.com/containers/podman/issues/10625. So we can(should?!) add the build tags to the shim package as well to make it consistent with the rest.